mlir/tools/rocmlir-tuning-driver/rocmlir-tuning-driver.cpp

```diff
@@ -41,27 +41,8 @@
 #include "llvm/Support/SourceMgr.h"
 #include <cstdlib>
 
-#include <hip/hip_runtime.h>
-
-#if !defined(_HIP_CLANG_ONLY__)
-// GCC complains if we don't do this
-template <std::size_t n, typename... Ts,
-          typename std::enable_if<n == sizeof...(Ts)>::type * = nullptr>
-void pArgs(const std::tuple<Ts...> &, void *) {}
-
-template <std::size_t n, typename... Ts,
-          typename std::enable_if<n != sizeof...(Ts)>::type * = nullptr>
-void pArgs(const std::tuple<Ts...> &formals, void **_vargs) {
-  using T = typename std::tuple_element<n, std::tuple<Ts...>>::type;
-
-  static_assert(!std::is_reference<T>{},
-                "A __global__ function cannot have a reference as one of its "
-                "arguments.");
-  _vargs[n] =
-      const_cast<void *>(reinterpret_cast<const void *>(&std::get<n>(formals)));
-  return pArgs<n + 1>(formals, _vargs);
-}
-#endif
+// Utilities to allocate buffers
+#include "../utils/performance/common/benchmarkUtils.h"
```

Comment:
Can we export these somewhere? (so we don't have a relative path)

Reply:
Nah, these are internal headers

Reply:
... on the other hand, this could go to `mlir/include/utils/`, perhaps.